client/my-sites/site-settings/theme-enhancements/index.jsx

```diff
@@ -139,27 +135,28 @@ class ThemeEnhancements extends Component {
 					disabled={ formPending }
 					/>
 
-				{
-					minilevenModuleActive && (
-						<div className="theme-enhancements__module-settings is-indented">
-							{
-								this.renderToggle( 'wp_mobile_excerpt', translate(
-									'Show excerpts on front page and on archive pages instead of full posts'
-								) )
-							}
-							{
-								this.renderToggle( 'wp_mobile_featured_images', translate(
-									'Hide all featured images'
-								) )
-							}
+				<div className="theme-enhancements__module-settings is-indented">
+					{
+						this.renderToggle( 'wp_mobile_excerpt', ! minilevenModuleActive, translate(
+							'Show excerpts on front page and on archive pages instead of full posts'
+						) )
+					}
+					{
+						this.renderToggle( 'wp_mobile_featured_images', ! minilevenModuleActive, translate(
+							'Hide all featured images'
+						) )
+					}
+					{
+						this.renderToggle( 'wp_mobile_app_promos', ! minilevenModuleActive, translate(
+							'Show an ad for the {{link}}WordPress mobile apps{{/link}} in the footer of the mobile theme',
```

Comment:
@yoavf Does this count as a new string to be translated or will the current translations still work for it?